springboard.yaml

```diff
@@ -43,7 +43,7 @@ models:
         type: string
 repositories:
```

Comment:
My idea was to update springboard.yaml to contain the production repo urls once they exist. Is there a reason not to do that instead?